query/src/configs/config.rs

```diff
@@ -113,33 +113,30 @@ pub struct LogConfig {
 
 impl LogConfig {
     pub fn default() -> Self {
-        LogConfig {
-            log_level: "INFO".to_string(),
-            log_dir: "./_logs".to_string(),
-        }
+        <Self as StructOpt>::from_iter(&Vec::<&'static str>::new())
     }
 }
 
 /// Store config group.
 /// serde(default) make the toml de to default working.
 #[derive(Clone, serde::Serialize, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
 pub struct StoreConfig {
-    #[structopt(long, env = STORE_ADDRESS, default_value = "", help = "Store backend address")]
+    #[structopt(long, env = STORE_ADDRESS, default_value, help = "Store backend address")]
     #[serde(default)]
     pub store_address: String,
 
-    #[structopt(long, env = STORE_USERNAME, default_value = "", help = "Store backend user name")]
+    #[structopt(long, env = STORE_USERNAME, default_value = "root", help = "Store backend user name")]
     #[serde(default)]
```

Comment:
`structopt`'s `default_value` is `""` while we set `store_username` to `root` in `StoreConfig::default()`. Which shall we choose?

Reply:
`default_value = "root"` is better here, show a default user for demo.